src/libgit2/transports/httpclient.c

```diff
@@ -7,7 +7,36 @@
 
 #include "common.h"
 #include "git2.h"
-#include "http_parser.h"
+
+#define USE_LLHTTP 1
```

Comment:
Leftover from devel testing?  This is also probably the cause of the CI failures, although CI seems to use the bundled dependencies so it may not really test how this builds with llhttp.

Reply:
Yeah, that was a glaring mistake.